Framework/Kernel/test/V2DTest.h

```diff
@@ -107,6 +107,19 @@ class V2DTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(p1, V2D(9.0, 27.0));
   }
 
+  void test_Negate_Gives_Same_Length_But_Opposite_Direction() {
+    const V2D p1(-3, 9);
+    const V2D p2 = -p1;
+    TS_ASSERT_EQUALS(p2, V2D(3, -9))
+  }
+
+  void test_Negate_Works_With_Special_Values() {
+    const V2D p1(INFINITY, std::nan(""));
```

Comment:
`std::nan("")` is correct (no need to change it), but I find [`std::numeric_limits<double>::quiet_NaN()`](http://en.cppreference.com/w/cpp/types/numeric_limits/quiet_NaN) is a bit clearer.

Reply:
I didn't know this exists in `std::numeric_limits`. Thanks for the hint!